docs/self-managed/zeebe-deployment/exporters/camunda-exporter.md

```diff
@@ -0,0 +1,189 @@
+| replicasByIndexName   | A map where the key is the index name and the value is the number of replicas, allowing you to override the default `numberOfReplicas` setting for specific indices.                        |         |
+
+</TabItem>
+
+<TabItem value="bulk">
+
+To avoid too many expensive requests to the Elasticsearch/Opensearch cluster, the exporter performs batch
+updates by default. The size of the batch, along with how often it should be flushed (regardless of
+size) can be controlled by configuration.
+
+| Option | Description                                                                                                                                                    | Default |
+| ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| delay  | Delay, in seconds, before force flush of the current batch. This ensures that even when we have low traffic of records, we still export every once in a while. | `5`     |
+| size   | The amount of records a batch should have before we flush the batch                                                                                            | `1000`  |
+
+With the default configuration, the exporter will aggregate records and flush them to Elasticsearch/Opensearch:
+
+1. When it has aggregated 1000 records. 2. Five seconds have elapsed since the last flush (regardless of how many records were aggregated).
+
+</TabItem>
+
+<TabItem value="retention">
+
+A retention policy can be set up to delete old data.
+When enabled, this creates an Index Lifecycle Management (ILM) Policy that deletes the data after the specified
+`minimumAge`.
+All index templates created by this exporter apply the created ILM Policy.
+
+| Option     | Description                                                                  | Default |
+| ---------- | ---------------------------------------------------------------------------- | ------- |
+| enabled    | If `true` the ILM Policy is created and applied to the index templates       | `false` |
+| minimumAge | Specifies how old the data must be, before the data is deleted as a duration | `30d`   |
+| policyName | The name of the created and applied ILM policy                               |         |
+
+:::note
+The duration can be specified in days `d`, hours `h`, minutes `m`, seconds `s`, milliseconds `ms`, and/or nanoseconds
+`nanos`.
+:::
+
+</TabItem>
+</Tabs>
+
+## Example
+
+Here is an example configuration of the exporter:
+
+```yaml
+---
+exporters:
+  # Camunda Exporter ----------
+  # An example configuration for the camunda exporter:
+  #
+  # These setting can also be overridden using the environment variables "ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_..."
```

Comment:
🔧 Worth to mention how the args look like, because this often lead to confusion, as this is not with `_`

Reply:
can youn explain a bit more what do you mean?

Reply:
Was thinking about an example of setting something in the ARGS section like:

`ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_ARGS_INDEX_NUMBEROFSHARDS`

For example related support case. https://jira.camunda.com/browse/SUPPORT-23300

Reply:
I added a note on how to convert it and an example below

Reply:
Double underscore?

Reply:
oh no, that's wrong. I guess I relied on co-pilot completing the sentence and didn't check for correctness 😄

Reply:
fixed